### PR TITLE
test: add webpay plus mall tests

### PR DESCRIPTION
**Pull request by isabellaahc**
This PR include:

- [x] unit tests for webpay plus mall transaction class
image

image

### CODE REVIEW

**Review by tribiec — APPROVED** [review]

File: tests/webpay/plus/test_mall_transaction.py [hunk 1]
```diff
@@ -1,82 +1,369 @@
...
+        with self.assertRaises(TransbankError) as context:
+            self.transaction.create(self.mall_buy_order_mock+'too_long', self.session_id_mock, self.return_url_mock,
+                                    self.get_mall_transaction_details())
+
+        self.assertTrue("'buy_order' is too long, the maximum length" in context.exception.message)
+        self.assertEqual(context.exception.__class__, TransbankError)
+
+    def test_create_mall_exception_session_id_max_length(self):
+        with self.assertRaises(TransbankError) as context:
+            self.transaction.create(self.mall_buy_order_mock, self.token_mock, self.return_url_mock,
```
Comment (Matiasnickolas) [inline tests/webpay/plus/test_mall_transaction.py:85]: for a better understanding of test, use a descriptive argument name. In this case, you use an argument named token as session_id , and the test is related to session_id

File: tests/webpay/plus/test_mall_transaction.py [hunk 2]
```diff
@@ -1,82 +1,369 @@
...
+            self.transaction.create(self.mall_buy_order_mock, self.session_id_mock, too_long_url,
+                                    self.get_mall_transaction_details())
+
+        self.assertTrue("'return_url' is too long, the maximum length" in context.exception.message)
+        self.assertEqual(context.exception.__class__, TransbankError)
+
+    def test_create_mall_exception_child_buy_order_max_length(self):
+        with self.assertRaises(TransbankError) as context:
+            self.transaction.create(self.mall_buy_order_mock, self.session_id_mock, self.return_url_mock,
+                                    MallTransactionCreateDetails(self.amount1_mock, self.child1_commerce_code,
```
Comment (Matiasnickolas) [inline tests/webpay/plus/test_mall_transaction.py:104]: same here

**Review by Matiasnickolas — APPROVED** [review]
LGTM 🤝

**Review by mvarlic — APPROVED** [review]
LGTM

**Pull request merged**